pyproject.toml

```diff
@@ -29,7 +29,7 @@ azure-storage-queue = "12.9.0"
 beautifulsoup4 = "4.12.3"
 fake-useragent = "1.5.1"
 chardet = "5.2.0"
-azure-search-documents = "11.4.0"
+azure-search-documents = "11.6.0b1"
```

Comment:
It is needed for Integrated Vectorisation

Reply:
Is it possible to use the latest beta version? It will probably be closest to the next stable version

Reply:
I tested this (0b1) version is working well for both Push & Pull. Will raise another PR after testing again in case we need to update to the latest beta version. @cecheta thoughts? Can I resolve conversation?

Reply:
Yeah of course 😄